app/(app)/settings/index.tsx

```diff
@@ -92,10 +101,11 @@ const SettingsPage = () => {
       <Modal visible={isModalVisible} onRequestClose={toggleModal}>
         <ModalContentWithActions
           variant="error"
+          isLoading={isPending}
           title={t('deleteAccountConfirmModal.title')}
           text={t('deleteAccountConfirmModal.message')}
           primaryActionLabel={t('deleteAccountConfirmModal.actionConfirm')}
-          primaryActionOnPress={handleConfirmDelete}
+          primaryActionOnPress={mutate}
```

Comment:
If used like this, I would like to see some more specific name than `mutate`. e.g. `mutateSignOut`?